src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php

```diff
@@ -54,26 +54,26 @@ public function __construct(TranslatorInterface $translator)
     public function getCollection()
     {
         $fields = [
-            new EntityField('id', $this->trans('ID', 'Admin.Global')),
-            new EntityField('active', $this->trans('Active (0/1)')),
-            new EntityField('name', $this->trans('Name', 'Admin.Global')),
-            new EntityField('address1', $this->trans('Address', 'Admin.Global'), '', true),
-            new EntityField('address2', $this->trans('Address (2)')),
+            new EntityField('id', $this->trans('ID', 'Admin.Global', 'Admin.Advparameters.Feature')),
```

Comment:
This one seems weird.. I think it's the second parameters array which is missing

```suggestion
            new EntityField('id', $this->trans('ID', [], 'Admin.Global')),
```

Reply:
My bad, there is no array parameter in this case, however the domain parameter was already provided

Reply:
yep, because of https://github.com/PrestaShop/PrestaShop/blob/7ab0cae901f6b932414415fa173c5802818f6ee6/src/Core/Import/EntityField/Provider/StoreContactFieldsProvider.php#L92 (in case someone see your comments)